src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/SearchByGEController.java

```diff
@@ -54,5 +59,47 @@ public String search(@RequestParam(name = "college", required = true) String col
         return "search/byge/results";
     }
     
+    
+    @GetMapping("/search/byge/multiquarter")
+    public String instructor(Model model, SearchByGEMultiQuarter SearchByGEMultiQuarter) {
```

Comment:
I don't believe this is necessary (be sure to test it though!)

```suggestion
    public String instructor(Model model) {
```